src/components/layout/InvestNav/index.tsx

```diff
@@ -9,7 +9,7 @@ import { selectUserStreams } from 'store/main/selectors';
 import gift from 'assets/images/gift.svg';
 import styles from './styles.module.scss';
 
-const TUTORIAL_LINK = 'https://docs.ricochet.exchange/tutorial/using-the-dapp';
+const TUTORIAL_LINK = 'https://docs.ricochet.exchange/';
 const RICOCHET_V1_LINK = 'https://v1.ricochet.exchange/';
 const SUPPORT = 'https://discord.com/channels/862796510604296263/864667072357597185';
```

Comment:
Isn't there a constants file somewhere you could put all the constants? The discord link is not a constant right now and is hard coded in two places, can you add DISCORD_INVITE_LINK too this so all the constants are in one place?